README.rst

```diff
@@ -73,7 +73,7 @@ Once you have that, you can run the tests with::
 
 You can specify what test(s) to run::
 
-	S3TEST_CONF=your.conf ./virtualenv/bin/nosetests s3tests.functional.test_s3:test_object_acl_grant_public_read
+	S3TEST_CONF=your.conf ./virtualenv/bin/nosetests s3tests.functional.test_s3:test_bucket_acl_grant_userid_read
```

Comment:
:+1: although could you pick a simpler test case like `test_bucket_list_empty` which is more likely to run correctly?  This test requires less user configuration and some S3 clones lack support for ACLs.